### DISCUSSION TITLE
Submission

### DISCUSSION THREAD

**Discussion by jv426** [Part-II-Computational-Physics/.github#16] [General]
Hello!

I have done the code by copying the code from github into a jupyter notebook and then coded there (linking the notebook with Github did not work out for me). Then I copied the code back into Github by going to "open in github.dev" and pasted all the code there in the right places. I then commited changes, but I'm paranoid that the changes were not saved in the main file. Does this procedure work (is the code properly saved and shared)?

**Comment by NathanMagnan** [comment]
Hello!

I've never done that procedure, so I don't know. My advice would be to:
- copy your file somewhere safe on your machine
- go to your repository on github and look at what's in there. Is the jupyter notebook in the original version or your modified version?
- If it's in the original version, it means your procedure doesn't work. In which case, follow the normal procedure (clone the online repository onto your machine , modify the cloned jupyter notebook, `git commit`, and finally `git push`)

**Reply by jv426** [reply]
Hello again,

I have checked and it seems that the modified (with my code) version is in my repository exercise-1-pdes-jv426. Would it be too much to ask you to check if you can also see the changes? Alternatively, can I upload an additional file into the repository as I have seen some other students have. I'm sorry for the inconvenience but this is my first time working with Github and I would like to get it right.

**Reply by NathanMagnan** [reply]
I can also see the changes

**Reply by jv426** [reply]
Thank you very much!
